subprojects/dependency-management/src/main/java/org/gradle/internal/resolve/ModuleVersionResolveException.java

```diff
@@ -37,7 +37,7 @@
 @Contextual
 public class ModuleVersionResolveException extends DefaultMultiCauseExceptionNoStackTrace {
     private final List<List<? extends ComponentIdentifier>> paths = new ArrayList<>();
-    private final ComponentSelector selector;
+    protected final ComponentSelector selector;
```

Comment:
💭 Probably better as a `private` field + a getter for the subtype, since this class isn't abstract.

Reply:
Fun fact, the method was already there :)